test/integration/auth.spec.ts

```diff
@@ -669,7 +669,7 @@ describe('admin.auth', () => {
     describe('SAML management APIs', () => {
       let tenantAwareAuth: admin.auth.TenantAwareAuth;
       const authProviderConfig = {
-        providerId: 'saml.' + generateRandomString(5),
+        providerId: 'saml.' + generateRandomString(10, false).toLowerCase(),
```

Comment:
Lets add a couple of helper methods and remove this duplication. `randomSamlProviderId()` and `randomOidcProviderId()`.

Reply:
done